runtime/ts/storage/storage-provider-factory.ts

```diff
@@ -20,9 +21,11 @@ export class StorageProviderFactory {
   constructor(private readonly arcId: Id) {
     // TODO: Pass this factory into storage objects instead of linking them directly together.
     // This needs changes to the StorageBase API to facilitate the FirebaseStorage.open functionality.
+    const inmemory = new InMemoryStorage(arcId);
     const firebase = new FirebaseStorage(arcId);
+    const pouchdb = new PouchDbStorage(arcId);
     const synthetic = new SyntheticStorage(arcId, firebase);
-    this._storageInstances = {'in-memory': new InMemoryStorage(arcId), firebase, synthetic};
+    this._storageInstances = {'in-memory': inmemory, firebase, synthetic, pouchdb};
```

Comment:
(quibble: inmemory -> inMemory)

Reply:
The perils of adding dashes to your scheme :)